apps/src/netsim/NetSimSendWidget.js

```diff
@@ -48,30 +81,137 @@ NetSimSendWidget.createWithin = function (element, connection) {
 };
 
 /**
+ * Creates a keyPress handler that allows only the given characters to be
+ * typed into a text field.
+ * @param {RegExp} whitelistRegex
+ * @return {function} appropriate to pass to .keypress()
+ */
+var whitelistCharacters = function (whitelistRegex) {
```

Comment:
Not sure if/what the style guide says, but in general I use the form function whitelistCharacter(whiteListRegex) {} when I'm a top level function, and the form var whitelistCharaccters = function (whitelistRegex) { } when I'm creating a function within another function (at which point it feels more like a variable to me)